tests/test_dmlscheduler.py

```diff
@@ -11,13 +11,14 @@
 from core.utils.dmljob          import DMLJob
 from core.scheduler             import DMLScheduler
 from core.configuration         import ConfigurationManager
-
+from core.communication_manager import CommunicationManager
 
 config_manager = ConfigurationManager()
 config_manager.bootstrap(
     config_filepath='tests/artifacts/runner_scheduler/configuration.ini'
 )
-scheduler = DMLScheduler(config_manager)
+communication_manager = CommunicationManager(config_manager)
+scheduler = DMLScheduler(config_manager, communication_manager)
```

Comment:
I'm curious to know how this doesn't fail if `CommunicationManager` doesn't get configured with the `scheduler`...